HHS/Head-Start-TTADP
# CD: Run database migrations

**Description of change**
Run database migrations as part of cd pipeline

**How to test**
- To verify migrations were run against sandbox db, follow new directions for "Interacting with a deployed database" in README.md. Then run `\d` psql command to verify that tables exist.
- After merge to main branch, verify migrations were run against dev db by following the above.  You can also login to cloud.gov and check the log stream for the "tta-smarthub-dev" application. The output from `yarn db:migrate:prod` will be viewable there.

**Issue(s)**
* https://github.com/HHS/Head-Start-TTADP/issues/202

**Checklist**

- [n/a] Meets accessibility standards (WCAG 2.1 Levels A, AA)
    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
